qucs/qucs/dialogs/aboutdialog.cpp

```diff
@@ -0,0 +1,236 @@
+    tr("German by Stefan Jahn"),
+    tr("Polish by Dariusz Pienkowski"),
+    tr("Romanian by Radu Circa"),
+    tr("French by Vincent Habchi, F5RCS"),
+    tr("Portuguese by Luciano Franca, Helio de Sousa, Guilherme Brondani Torri"),
+    tr("Spanish by Jose L. Redrejo Rodriguez"),
+    tr("Japanese by Toyoyuki Ishikawa"),
+    tr("Italian by Giorgio Luparia and Claudio Girardi"),
+    tr("Hebrew by Dotan Nahum"),
+    tr("Swedish by Markus Gothe and Peter Landgren"),
+    tr("Turkish by Onur and Ozgur Cobanoglu"),
+    tr("Hungarian by Jozsef Bus"),
+    tr("Russian by Igor Gorbounov"),
+    tr("Czech by Marek Straka and Martin Stejskal"),
+    tr("Catalan by Antoni Subirats"),
+    tr("Ukrainian by Dystryk"),
+    tr("Arabic by Chabane Noureddine"),
+    tr("Kazakh by Erbol Keshubaev")
+  }};
+  
+
+  std::shuffle(currAuths.begin(), currAuths.end(), rng);
+
+  QLabel *lbl;
+
+  setWindowTitle(tr("About Qucs"));
+
+  all = new QVBoxLayout(this);
+  //all->setContentsMargins(0,0,0,0);
+  //all->setSpacing(0);
+
+  QLabel *iconLabel = new QLabel();
+  iconLabel->setPixmap(QPixmap(QString(":/bitmaps/hicolor/128x128/apps/qucs.png")));
+
+  QWidget *hbox = new QWidget();
+  QHBoxLayout *hl = new QHBoxLayout(hbox);
+  hl->setContentsMargins(0,0,0,0);
+
+  hl->addWidget(iconLabel);
+  all->addWidget(hbox);
+ 
+  QWidget *vbox = new QWidget();
+  QVBoxLayout *vl = new QVBoxLayout(vbox);
+  //vl->setContentsMargins(0,0,0,0);
+  hl->addWidget(vbox);
+
+  QString versionText;
+  versionText = tr("Version")+" "+PACKAGE_VERSION+
+#ifdef GIT
+    " ("+GIT+") " +
+#endif
+    "\n";
+
+  vl->addWidget(new QLabel("<span style='font-size:x-large; font-weight:bold;'>Quite Universal Circuit Simulator</span>"));
+  lbl = new QLabel(versionText);
+  lbl->setAlignment(Qt::AlignHCenter);
+  vl->addWidget(lbl);
+  vl->addWidget(new QLabel(tr("Copyright (C)")+" 2003-2009 "+
+			   tr("by Michael Margraf")+"\n"+
+			   tr("Copyright (C)")+" 2011-2015 "+
+			   tr("Qucs Team")));
+
+  lbl = new QLabel("\nThis is free software; see the source for copying conditions."
+		   "\nThere is NO warranty; not even for MERCHANTABILITY or "
+		   "\nFITNESS FOR A PARTICULAR PURPOSE.\n");
+  lbl->setAlignment(Qt::AlignHCenter);
+  all->addWidget(lbl);
+
+  QTabWidget *t = new QTabWidget();
+  all->addWidget(t);
+  connect(t, SIGNAL(currentChanged(int)), this, SLOT(currentChangedSlot(int)));
+  
+
+  authorsBrowser = new QTextBrowser;
+  trBrowser = new QTextBrowser;
+
+
+
+  QString supportText;
+  // link to home page, help mailing list, IRC ?
+  supportText = tr("Home Page") + " : <a href='http://qucs.sourceforge.net/'>http://qucs.sourceforge.net/</a><br/>"+
+    tr("Documentation start page") + " : <a href='http://qucs.sourceforge.net/docs.html'>http://qucs.sourceforge.net/docs.html</a><br/>" +
+    tr("Components reference manual") + " : <a href='http://qucs.sourceforge.net/doc/0.0.19/html/index.html'>http://qucs.sourceforge.net/doc/0.0.19/html/index.html</a><br/>" +
```

Comment:
This link is dead.
I moved the component reference to https://github.com/guitorri/qucs-manual. Perhaps we can move it to the Qucs organization. It is still needs some work thou.
Perhaps the [Readme.md resources](https://github.com/Qucs/qucs#resources) would fit here nicely.